### DISCUSSION TITLE
Little confusion

### DISCUSSION THREAD

**Discussion by mahdihasan2006** [anyproto/contributors#240] [Q&A] [answered]
One default object type "Contact" has a relation "Phone" with relation type "Phone". If I want to add multiple phone numbers, how should I do that?

1. Write multiple phone numbers in the input field of phone relation like this: +123, +432, +387
2. Create multiple relations like "Phone 1", "Phone 2", "Phone 3"
3. Or, use "Multi-select" as relation type

**Comment by filip-anytype** [comment, marked as answer]
You can use any of the three you want (and I'm sure there are more ways to do it). Each one of these has its pros and cons, so it really does depend on your specific use case. For instance, if you use the first approach, you will not be able to easily copy numbers, or call someone directly from the relation since the relation only expects one value.
